.gemini/config.yaml

```
@@ -17,4 +17,4 @@ ignore_patterns:
   - bin/internal/*.version
   - engine/src/flutter/ci/licenses_golden/**
   # Avoid code reviews on all third_party files.
-  - **/third_party/**
+  - */third_party/*
```

I think this will still have problems parsing, as YAML treats it as an [alias to an anchor](https://yaml.org/spec/1.2.2/#71-alias-nodes).

I'd just make the pattern an explicit string:

```suggestion
  - "**/third_party/**"
```

---

I poked around a bit on GitHub and found a repo with a `**/...` pattern in quotes where GCA was working, so it seems like this is the way.

---

Thank you, @parlough!